### DISCUSSION TITLE
Use AI/Machine Learning to improve Version Tracking score/confidence

### DISCUSSION THREAD

**Discussion by astrelsky** [NationalSecurityAgency/ghidra#5267] [Ideas]
I think it would be neat to use AI or machine learning to improve the score and confidence values of results in version tracking. This could be done in a similar fashion as the random forest function finder and have it trained off of already accepted matches.

**Additional context**
AI and machine learning is still black magic to me and I have no idea what's what or how applicable it would really be here.

**Comment by ryanmkurtz** [comment]
I converted this to a discussion so other people can weigh in and add their ideas.  Perhaps it will evolve into something actionable.
